keel-sql/src/main/resources/db/changelog/20201008-rename-delivery-artifact.yml

```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: rename-delivery-artifact
+      author: lpollo
+      changes:
+        - renameTable:
```

Comment:
I'd like us to get in the happy of doing database migrations that don't generate errors. 

As implemented, when this migration takes effect during the red-black deployment, it's going to cause the old ASG to throw errors because it will be querying a database that no longer exists.

Here's one possible strategy:

Step 1: Create the new table, modify the write logic to write to both tables, modify the read the logic to read from the new table first, then read from the old one if the data is not in the new one.

Step 2:  Once the code is no longer running, backfill the new table with data from the old one.

Step 3: Modify the code to only read/write the new table, write a migration to drop the old table.

Reply:
Also, if you were going to rename the database table, I would start a new PR for step 1 rather than rolling it into this one.